### PR TITLE
Feature/DTGB-507: Fixed some issues.

### PR DESCRIPTION
**Pull request by meirege**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated the style guide CHANGELOG accordingly.
- [ ] I have updated the gent_base CHANGELOG accordingly.

### CODE REVIEW

File: styleguide/components/21-atoms/button/_button.scss [hunk 1]
```diff
@@ -149,14 +149,13 @@
...
   justify-content: space-between;
-  padding: 0 1.2em;
+  padding: 0 .8em 0 1.2em;
   line-height: 1.6;
   text-align: left;
 
   &::before {
     order: 2;
     margin-left: .5rem;
-    padding-right: .2rem;
```
Comment (Jeroen005) [inline styleguide/components/21-atoms/button/_button.scss:159]: gecheckt dat de animatie niet springt als je hovert?

File: styleguide/components/31-molecules/teasers/teaser.scss [hunk 2]
```diff
@@ -1,16 +1,18 @@
 .teaser {
-
   position: relative;
   list-style: none;
 
   figure {
+    &:only-child {
```
Comment (Jeroen005) [inline styleguide/components/31-molecules/teasers/teaser.scss]: mss beter om alle childs gwn margin te geven? :) fixed ook het issue voor .tag dan.

**Review by Jeroen005 — CHANGES_REQUESTED** [review]

File: styleguide/components/21-atoms/button/_button.scss (hunk 1 above)
Comment (meirege) [reply styleguide/components/21-atoms/button/_button.scss:159]: Ziet er goed uit volgens mij :)

File: styleguide/components/31-molecules/teasers/teaser.scss (hunk 2 above)
Comment (meirege) [reply styleguide/components/31-molecules/teasers/teaser.scss]: Goed punt.

**Review by Jeroen005 — APPROVED** [review]

**Pull request merged**